django_celery_fulldbresult/models.py

```diff
@@ -1,12 +1,23 @@
 from __future__ import absolute_import, unicode_literals
+
+import json
+
+from django.conf import settings
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
 
+try:
+    from django.utils.encoding import force_text
+except ImportError:
+    from django.utils.encoding import force_unicode as force_text  # noqa
```

Comment:
Not sure about using external tools' flags in the code for an open-sourced project but it might just be me feeling extra-nitpicky at the moment :-)